tests/core/test_summary.py

```diff
@@ -31,19 +30,19 @@ def test_steplist(gcd_with_metrics, capfd):
 
     gcd_with_metrics.summary()
     stdout, _ = capfd.readouterr()
+    print(stdout)
```

Comment:
is this left over?

Reply:
I think this should stay - this test captures stdout to check the output of summary(), but that also hides it. It's useful to print back out for debugging - I added an explanatory comment.